### DISCUSSION TITLE
Support Lit code generator

### DISCUSSION THREAD

**Discussion by vini-brito** [teleporthq/teleport-code-generators#708] [Q&A]
Hello! On adding Lit as a code generator, I take it would be similar to standard HTML, any pointers on doing this?
Reference on Lit: https://lit.dev/

**Comment by paulbrie** [comment]
Hi! Thanks for reaching out. For now we do not plan to support new languages. If the demand increases however, we can reconsider this. Best, Paul

**Comment by vini-brito** [comment]
Yeah, sure, but how could I go about adding it myself, if possible? I looked into the JSON to element generator but I am unsure if I found the complete docs about it, hence me asking for some other pointer if available (:

**Reply by paulbrie** [reply]
Hi!

Here are our docs (not completely updated) but which will help: https://docs.teleporthq.io/guides/getting-started.html.
To fiddle around you can use this: https://repl.teleporthq.io/

If you are really interested into creating one please join our Discord server and we could have a chat in order to provide you with the right level of support https://discord.com/invite/uJrbeuHhyR
